### PR TITLE
Adds support to javascript async functions

### PR DESCRIPTION
**Pull request by marcelo140**
Fixes the issue with the javascript async keyword as described in https://github.com/universal-ctags/ctags/issues/1537.

### CODE REVIEW

File: parsers/jscript.c [hunk 1]
```diff
@@ -226,8 +227,9 @@ static const keywordTable JsKeywordTable [] = {
 	{ "class",		KEYWORD_class				},
 	{ "extends",	KEYWORD_extends				},
 	{ "static",		KEYWORD_static				},
-	{ "default",		KEYWORD_default				},
+	{ "default",	KEYWORD_default				},
 	{ "export",		KEYWORD_export				},
+    { "async",      KEYWORD_async               },
```
Comment (masatake) [inline parsers/jscript.c]: It seems that one extra space is put here. Could you remove it?

**Comment by masatake** [comment]
Please, ignore the result of circleci.

**Comment by marcelo140** [comment]
Replaced the spaces I introduced with tabs.

**Comment by masatake** [comment]
#1544 solves the original issue.
Thank you for making pull request. This patch drives @b4n:-)

**Pull request closed without merging**